tensorflow/python/ops/array_ops.py

```diff
@@ -259,11 +259,8 @@ def rank_internal(input, name=None, optimize=True):
       return gen_array_ops.rank(input, name=name)
 
 
-# DEPRECATED use init_ops.zeros_initializer
-# TODO(irving) Move it to init_ops.py
-def zeros_initializer(shape, dtype=dtypes.float32, partition_info=None):
-  """An adaptor for zeros() to match the Initializer spec."""
-  return zeros(shape, dtype)
+#DEPRECATED
+zeros_initializer = init_ops.zeros_initializer
```

Comment:
Ok, at least I changed all reference of it. So, it can work now. I'll revert the latest commit